docs/developer-docs/daos/nns/concepts/neurons/neuron-overview.mdx

```diff
@@ -46,34 +46,49 @@ In addition, each neuron has an associated **account** on the ICP ledger where t
 ### Neuron permissions
 The **controller** is a principal that controls the neuron and which cannot be changed. A principal corresponds to the public key of a public-private key pair, and anyone with possession of the corresponding private key will have full control of the neuron. Therefore, the private key of the controller principal should be kept very secure. A principal can control many neurons.
 
-In addition to the controller, a neuron can have **hotkeys**. Hotkeys are defined by a list of principal ID and can be used to perform actions with limited privileges, namely voting, following (see below), and reading the full information about the neuron.
+### Hotkeys
+In addition to the controller, a neuron can have **hotkeys**. Hotkeys are principal IDs that can perform actions with limited privileges, namely:
+
+- Voting.
+
+- Following (see below).
+
+- Reading the full information about the neuron, including non-public fields.
+
+Hotkeys cannot increase dissolve delay, start dissolving, or disburse a neuron's stake.
```

Comment:
There are more things a hotkey can't do (change visibility, (auto) stake maturity, join neuron's fund, come to mind. Should this be an exhaustive list or should it be made clear that it's just some examples?

Reply:
Revised per Lara's suggestion